packages/node_modules/samples/browser-call-with-screenshare/test/wdio/spec/normal-dialing.js

```diff
@@ -48,18 +48,14 @@ describe('samples', () => {
       it('starts screensharing', () => {
         browser.pause(2500);
         browserSpock.click('button[title="share screen"]');
+
+        browserSpock.waitUntil(() => (browserSpock.getText('#screenshare-tracks') === '1'), 10000, 'Timed-out waiting for screenshare tracks to chage');
       });
 
       it('stops screensharing', () => {
         browser.pause(2500);
         browserSpock.click('button[title="stop screen share"]');
```

Comment:
Are we still hiding some buttons? I know that was causing issues before. I like that this waits for an element to update.

Reply:
Nothing should be hiding that I'm aware of.